filament/src/PostProcessManager.cpp

```diff
@@ -316,10 +346,13 @@ FrameGraphResource PostProcessManager::ssao(FrameGraph& fg, FrameGraphResource d
                     SSAOPassData const& data, DriverApi& driver) {
                 auto depth = resources.getTexture(data.depth);
                 auto ssao = resources.getRenderTarget(data.ssao);
+                auto const& desc = resources.getDescriptor(data.ssao);
 
                 SamplerParams params;
                 FMaterialInstance* const pInstance = mSSAOMaterialInstance;
                 pInstance->setParameter("depth", depth, params);
+                pInstance->setParameter("resolution",
```

Comment:
Should we pass this to all materials instead of the surface dimensions like we do today?

Reply:
Yes, I think this will be part of the "post process material".